Networking/Networking/Extensions/Array+Woo.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+
+// MARK: - Array Helpers
+//
+extension Array where Element == Int {
+
+    /// Returns a de-duplicated array of integer values as a comma-separated String.
+    ///
+    func intToString() -> String {
```

Comment:
suggestion: it's hard to tell this helper deduplicates and sorts the array unless we look at the implementation, it might come as a surprise. Maybe some name like `sortedUniqueIntToString` would be more clear?